connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResource.java

```diff
@@ -166,6 +169,13 @@ public void destroyConnector(final @PathParam("connector") String connector) thr
         completeOrForwardRequest(cb, "/connectors/" + connector, "DELETE", null);
     }
 
+    @PUT
+    @Path("/{connectorType}/config/validate")
```

Comment:
How are `{connectorType}` here and `{connector}` differentiated? Couldn't they be mistaken for each other? Right now this probably works because only this path will match the suffix `/config/validate`, but I would worry about conflicts in the future (and the naming approach doesn't currently follow the normal REST layout where different resources will have a different prefix).

I think this is annoying here because we tend to treat "connector instance" and "connector plugin" as interchangeable, but in this case we don't want them to be.